packages/builder/src/components/design/PropertiesPanel/PropertyControls/OptionsEditor/OptionsEditor.svelte

```diff
@@ -10,9 +10,15 @@
   let drawer
   let tempValue = value || []
 
-  const saveFilter = async () => {
-    // Filter out incomplete options
-    tempValue = tempValue.filter(option => option.value && option.label)
+  const saveOptions = async () => {
+    // Filter out incomplete options, default if needed
+    tempValue = tempValue.filter(option => option.value || option.label)
+    for (let i = 0; i < tempValue.length; i++) {
+      let option = tempValue[i]
+      option.label = option.label ? option.label : option.value
+      option.value = option.value ? option.value : option.label
+      tempValue[i] = option
```

Comment:
NAB: you don't need to change anything, but just letting you know that you don't need to reassign `option` back to `tempValue[i]` here. When using objects JS will assign by reference rather than value, which means that after doing `let option = tempValue[i]`. both `option` and `tempValue[i]` are referencing the exact same object and mutating either will update the same value.